packages/wallet/test/integration/transactionTime.test.ts

```diff
@@ -11,7 +11,7 @@ describe('integration/transactionTime', () => {
   });
 
   it('provides utils necessary for computing transaction time', async () => {
-    const transactions = await firstValueFrom(wallet.transactions.history.incoming$);
+    const transactions = await firstValueFrom(wallet.transactions.history.all$);
```

Comment:
This is fine, but why was this change needed?

Reply:
`valueReceivedInspector` checks that the wallet addresses are within the transaction inputs to see if it's incoming.

Now `incoming$` is always empty as the inspector cannot match the addresses and inputs because the `createWallet` used for testing generates a new key agent with different address each run and the mocked wallet provider always returns the same transactions with hardcoded inputs and addresses.

Reply:
I see, but still not sure why it worked before. Wondering if we've changed how it works :thinking: